examples/e2e-tests/kubernetes/windows/hybrid/definition.json

```diff
@@ -21,7 +21,18 @@
         "count": 2,
         "vmSize": "Standard_D2_v2",
         "availabilityProfile": "AvailabilitySet",
-        "osType": "Windows"
+        "osType": "Windows",
+        "extensions": [

```

Comment:
Is this change related to the new test coverage?

Reply:
this file already has SSH enabled, so is there a need for WinRM too?